tests/test_emulate.py

```diff
@@ -82,10 +126,33 @@ def test_emulate_disasm_loop(start_binary):
         '   0x40009f                add    byte ptr [rax], al',
         '   0x4000a1                add    byte ptr [rax], al',
         '   0x4000a3                add    byte ptr [rax], al',
-        '   0x4000a5                add    byte ptr [rax], al'
+        '   0x4000a5                add    byte ptr [rax], al',
+    ]
+
+    disasm_without_emu_0x401000 = [
+        ' ► 0x401000 <_start>       movabs rsi, string                   <0x401014>',
+        '   0x40100a <_start+10>    mov    rdi, rsp',
+        '   0x40100d <_start+13>    mov    ecx, 3',
+        '   0x401012 <_start+18>    rep movsb byte ptr [rdi], byte ptr [rsi]',
+        '   0x401014 <string>       xor    dword ptr [rdx], esi',
+        '   0x401016 <string+2>     xor    esi, dword ptr [rsi]',
+        '   0x40101d                add    byte ptr [rax], al',
+        '   0x40101f                add    byte ptr [rax], al',
+        '   0x401021                add    byte ptr [rax], al',
+        '   0x401023                add    byte ptr [rax], al',
+        '   0x401025                add    byte ptr [rax], al',
     ]
 
-    assert nearpc(to_string=True) == disasm_without_emu
-    assert emulate(to_string=True, emulate=False) == disasm_without_emu
-    assert pdisass(to_string=True) == disasm_without_emu
-    assert u(to_string=True) == disasm_without_emu
+    compare_output_emu(disasm_with_emu_0x400080, disasm_with_emu_0x401000)
+    compare_output_without_emu(disasm_without_emu_0x400080, disasm_without_emu_0x401000)
+
+
+def compare_output_emu(emu_0x400080, emu_0x401000):
+    assert emulate(to_string=True) == emu_0x400080 or emu_0x401000
+
+
+def compare_output_without_emu(emu_0x400080, emu_0x401000):
+    assert nearpc(to_string=True) == emu_0x400080 or emu_0x401000
+    assert emulate(to_string=True, emulate=False) == emu_0x400080 or emu_0x401000
+    assert pdisass(to_string=True) == emu_0x400080 or emu_0x401000
+    assert u(to_string=True) == emu_0x400080 or emu_0x401000
```

Comment:
We compare the output with two expectations 0x400080 OR 0x401000.

Reply:
```
In [3]: assert 0 == 2 or 0
---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
<ipython-input-3-bac48926e1f1> in <module>
----> 1 assert 0 == 2 or 0
```

This needs to be changed to `assert X in (A, B)` or `assert X == A or X == B`

Reply:
It was solved in the discussion https://github.com/pwndbg/pwndbg/pull/868#discussion_r549111511